src/cl65/main.c

```diff
@@ -321,12 +317,8 @@ static void CmdAddFile (CmdDesc* Cmd, const char* File)
 {
     /* Expand the file vector if needed */
     if (Cmd->FileCount == Cmd->FileMax) {
-        unsigned NewMax   = Cmd->FileMax + 10;
-        char**   NewFiles = xmalloc (NewMax * sizeof (char*));
-        memcpy (NewFiles, Cmd->Files, Cmd->FileMax * sizeof (char*));
-        xfree (Cmd->Files);
-        Cmd->Files   = NewFiles;
-        Cmd->FileMax = NewMax;
+        Cmd->FileMax += 10;
+        Cmd->Files    = xrealloc(Cmd->Files, Cmd->FileMax * sizeof(char*));
```

Comment:
Please add the (unusual) spaces before the parameter lists.

Reply:
Done.